AppInspector/rules/default/components/active_content.json

```diff
@@ -161,37 +126,30 @@
       "cpp",
       "csharp"
     ],
-    "tags": [
-      "Component.Executable.Adobe.PDF"
-    ],
+    "tags":[ "Component.Executable.Adobe.PDF" ],
     "severity": "moderate",
     "patterns": [
       {
         "pattern": ".pdf",
```

Comment:
Maybe substring `pdf` instead? Not sure how common `pdf` would be in a string that has nothing to do with PDFs.

Reply:
That's definitely less precise.  I can easily imagine some variable name having pdf in it but not being a pdf.